src/Components/Pages/Home.jsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import '../Styles/Home.css';
+import Display from '../Utilities/Display';
+import VideoList from '../Utilities/VideoList';
+import { useVideo } from '../Providers/VideoProvider';
+
+const Home = () => {
+  const { videos } = useVideo();
+
+  return (
+    <main className='container'>
+      <Display />
+      <section className='video-listings flex-center flex-col align-start'>
+        <h1 className='h3' id='valorant'>
+          Valorant
+        </h1>
+        <VideoList
+          videos={videos.filter((video) => video._id.includes('valorant'))}
+        />
+        <h1 className='h3' id='csgo'>
```

Comment:
you can give a line space before starting a new category, to increase the readability.

And also, I think, it is good to have only one `<h1>` on a page for SEO purpose. You can read about this.